lib/ToastContainer.js

```diff
@@ -248,13 +248,19 @@ class ToastContainer extends Component {
                     pointerEvents="none"
                     ref={ele => this._root = ele}
                 >
+                {
+                    props.view ?
+                    <View>
+                        <InnerView />
+                    </View> :
                     <Text style={[
-                        styles.textStyle,
-                        props.textStyle,
-                        props.textColor && {color: props.textColor}
-                    ]}>
+                            styles.textStyle,
+                            props.textStyle,
+                            props.textColor && {color: props.textColor}
+                        ]}>
                         {this.props.children}
                     </Text>
+                }   
```

Comment:
{typeof children === 'string' ?
                        <Text style={[
                            styles.textStyle,
                            textStyle,
                            textColor && {color: textColor}
                            ]}
                        >
                            {children}
                        </Text> :
                        <View>
                            {children}
                        </View>
                    }

and now you do not need to set props